packages/editor-ui/src/components/RunData.vue

```diff
@@ -269,6 +269,13 @@ export default defineComponent({
 				!(this.binaryData && this.binaryData.length > 0)
 			);
 		},
+		showPinDataButton(): boolean {
```

Comment:
This could be written to be more readable. WDYT?

Reply:
Sure ofc yes